### ISSUE TITLE
Python Errors in Mailpile cli

### ISSUE THREAD

**Issue by TmCTux** [mailpile/Mailpile#2127]
Hello,

here is the issue observed:
when entered "m", "mail" or as bellow "mail test", it's probably not a good command and use of the cli but for someone starting using mailpile it is confusing to have those error reported directly in the interface.
«
mailpile> mail test
What message is test?
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mailpile/commands.py", line 584, in _run_sync
    return self._finishing(command(self, *args, **kwargs))
  File "/usr/lib/python2.7/dist-packages/mailpile/commands.py", line 582, in command
    return self.command(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/mailpile/plugins/compose.py", line 1035, in command
    return Update.command(self, create=create, outbox=outbox)
  File "/usr/lib/python2.7/dist-packages/mailpile/plugins/compose.py", line 999, in command
    email.update_from_string(session, update_string, final=outbox)
AttributeError: 'NoneType' object has no attribute 'update_from_string'

mail error: Failed: mail test
Elapsed: 0.029s (mail: Failed: mail test)

Aucun résultat
»

just interface improvement I guess.
thanks

**Comment by nik1082** [comment]
Hello! Is this issue still up? I'll try and work on it!

This is my first time contributing to an open-source repo, so bear with me! I've already written a short solution to this problem, however I could not test it. Running on Windows 7 Ultimate, both Python 3.7 and 2.7 are installed. I already made sure that the virtualenv is running Python 2.7 and followed all the instructions on your wiki, however, running `start .\mp.cmd` still yields the following errors:

```
(MP-VIR~1) C:\...\Mailpile>Traceback (most recent call last):
  File "scripts\mailpile", line 11, in <module>
    from mailpile.app import Main
  File "C:\...\Mailpile\mailpile\__init__.py", line 1, in <module>
    from mailpile.i18n import gettext as _
  File "C:\...\Mailpile\mailpile\i18n.py", line 5, in <module>
    from jinja2 import Environment, BaseLoader, TemplateNotFound
ImportError: No module named jinja2
```

To verify that the dependencies are installed, here is my `pip list`:

```
Package                       Version
----------------------------- ---------
appdirs                       1.4.3
asn1crypto                    0.24.0
astroid                       1.6.5
backports.functools-lru-cache 1.5
certifi                       2018.8.13
cffi                          1.11.5
colorama                      0.3.9
configparser                  3.5.0
cryptography                  2.3
enum34                        1.1.6
fasteners                     0.14.1
funcsigs                      1.0.2
futures                       3.2.0
idna                          2.7
imgsize                       2.1
ipaddress                     1.0.22
isort                         4.3.4
Jinja2                        2.10
lazy-object-proxy             1.3.1
lockfile                      0.12.2
lxml                          4.2.4
MarkupSafe                    1.0
mccabe                        0.6.1
mock                          2.0.0
monotonic                     1.5
nose                          1.3.7
pbr                           4.2.0
pgpdump                       1.5
Pillow                        5.2.0
pip                           18.0
pip-autoremove                0.9.1
pipenv                        2018.7.1
pycparser                     2.18
pycrypto                      2.6.1
pydns                         2.3.6
pylint                        1.9.3
pyreadline                    2.1
PySocks                       1.6.8
selenium                      3.14.0
setuptools                    40.0.0
singledispatch                3.4.0.3
six                           1.11.0
spambayes                     1.1b3
stem                          1.6.0
urllib3                       1.23
virtualenv                    16.0.0
virtualenv-clone              0.3.0
wheel                         0.31.1
win-inet-pton                 1.0.1
wrapt                         1.10.11
```

**Comment by nik1082** [comment]
Disregard the previous comment, already got it working somehow! Above is the PR :)

**Comment by Komish** [comment]
Looks like merged PR #2133 fixed this. Should it be closed?

**Comment by BjarniRunar** [comment]
I think so, thanks for the reminder @Komish!
